cmd/registries-operator/main.go

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  *
  */
+
+// Command main is a cli for this k8s operator
```

Comment:
i think for `golint` we don't need to comment the package name , so we can remove this

Reply:
remove this but failed on local...
```
$golint cmd/registries-operator/main.go
cmd/registries-operator/main.go:1:1: package comment should not have leading space
```

Reply:
ok i understand all
ref: https://github.com/golang/lint/issues/22

Reply:
ok i understand all.
ref: https://github.com/golang/lint/issues/22

Reply:
```golang

 * limitations under the License. */

package main

``